pfetch

```diff
@@ -367,6 +373,14 @@ get_uptime() {
             # regular seconds.
             s=$(($(system_time) / 1000000))
         ;;
+
+        SunOS)
+            read -r _ s <<-EOF
+                $(kstat -p unix:0:system_misc:snaptime)
```

Comment:
Full output of `kstat -p unix:0:system_misc:snaptime`?

Reply:
```
unix:0:system_misc:snaptime	14809.906993005
```